src/pages/components/accordion/code.mdx

```diff
@@ -1,22 +1,177 @@
 ---
 title: Accordion
-description: The accordion component delivers large amounts of content in a small space through progressive disclosure. The user gets key details about the underlying content and can choose to expand that content.
-tabs: ["Code", "Usage", "Style", "Accessibility"]
+description:
+  The accordion component delivers large amounts of content in a small space
+  through progressive disclosure. The user gets key details about the underlying
+  content and can choose to expand that content.
+tabs: ['Code', 'Usage', 'Style', 'Accessibility']
 ---
 
+import usage from './snippets/usage';
+import skeleton from './snippets/skeleton';
+
+<PageDescription>
+
+Learn how to build an accordion in React.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Usage</AnchorLink>
+<AnchorLink>Component API</AnchorLink>
+<AnchorLink>FAQ</AnchorLink>
+
+</AnchorLinks>
+
+## Usage
+
+You can build an accordion using a combination of the `Accordion` and
+`AccordionItem` components. The `Accordion` components acceps a list of
+`AccordionItem` components as `children` and each `AccordionItem` is responsible
+for displaying the accordion's heading and panel content.
+
+You can configure the accordion item's heading using the `title` prop.
+Everything you pass in as a child of `AccordionItem` will be rendered in the
+accordion's panel.
+
+<ComponentDemo links={{}}>{usage}</ComponentDemo>
+
+### Skeleton
+
+You can use the `AccordionSkeleton` component to render a skeleton variant of an
+accordion. This is useful to display while content in your accordion is being
+fetched from an external resource like an API.
+
 <ComponentDemo
-  links={{
-    React:
-      "http://react.carbondesignsystem.com/?path=/story/accordion--default",
-    Angular:
-      "https://angular.carbondesignsystem.com/?path=/story/components-accordion--basic",
-    Vue:
-      "http://vue.carbondesignsystem.com/?path=/story/components-cvaccordion--default",
-    Vanilla: "https://the-carbon-components.netlify.com/?nav=accordion",
-  }}
-  knobs={{ AccordionItem: ["open"] }}
->{`
+  links={{}}
+  scope={{ useState: React.useState, useEffect: React.useEffect }}
+  noInline>
+  {skeleton}
+</ComponentDemo>
+
+## Component API
+
+### Accordion props
+
+| Prop      | Type                     | Required |
+| --------- | ------------------------ | -------- |
+| align     | `'start'` &#124; `'end'` | `false`  |
+| children  | `node`                   | `true`   |
+| className | `string`                 | `false`  |
+
+Additional props passed into `Accordion` will be forwarded along to the
+underlying accordion container.
+
+### Accordion align
+
+In rare cases, you may need to specify the alignment of the icon in the
+accordion. You can use the `align` prop to specify the side where the icon
+should be placed.
+
+_Note: This prop must not be used to create a tree view or set of nested
```

Comment:
These note's might be better suited as [inline notifications](https://gatsby-theme-carbon.now.sh/components/InlineNotification). What do you think @aagonzales?

Reply:
@vpicone yeah, there was some chatter about it below where we talked about this.